### ISSUE TITLE
Ensure that the addon is valid

### ISSUE THREAD

**Issue by asbjornu** [PayEx/Prestashop-modules#6]
The addon should be validated with [PrestaShop's Addon Validator](https://validator.prestashop.com/) and all the problems listed there should be fixed. Currently, the validation fails with the following error:

> [The essential bootstrap lines are not respected](http://doc.prestashop.com/display/PS16/Creating+a+PrestaShop+module#CreatingaPrestaShopmodule-Creatingafirstmodule).
> [Using the backward compatibility toolkit](http://doc.prestashop.com/display/PS16/Using+the+backward+compatibility+toolkit).
> 
> ``` php
> $this->name = 'ssn' must have only alphanumeric characters (lowercase) & "_" (underscores)
> /ssn.php
> ```

**Comment by olegisk** [comment]
Yes. There are problems with validation. I think it's can be fixed.
